### ISSUE TITLE
Need to manually set thickness property when importing gltf scene

### ISSUE THREAD

**Issue by GitGhillie** [bevyengine/bevy#10931]
## Bevy version

0.12.1

## What you did

1. Create new principled BSDF material in Blender
2. Set transmission close to 1
3. Set IOR to something other than 1 to get some refraction going
4. Export gltf
5. Load the scene in Bevy using the following code:
```rust
commands.spawn(SceneBundle {
        scene: asset_server.load("scene.glb#Scene0"),
        ..default()
    });
```

## What went wrong

No refraction is seen when the scene is loaded in the Bevy app. Need to manually set the `thickness` property of the StandardMaterial to achieve a similar look to the Eevee renderer in Blender. The StandardMaterial documentation does not seem to include how to approach this exactly. But ideally it would work out of the box.

![transmission](https://github.com/bevyengine/bevy/assets/23659539/5186cc4f-422d-44da-8f9a-43e8915a0e7a)

## Additional information

Let me know if a reproducer would help, wanted to post first in case I just missed some export setting in Blender.

**Comment by alice-i-cecile** [comment]
A reproducer would be great. Once you have something small working; can you check if the property is correctly set in the gltF file? There's a chance this bug is on the Blender end, not the Bevy one.

**Comment by GitGhillie** [comment]
We had some discussion about the issue [⁠here.](https://discord.com/channels/691052431525675048/743663924229963868/1186578067406200902)
IMO we should ignore Blender for now and try to figure out the following:
How does one create a solid glass material in Bevy?

For example, if you have a Suzanne made of solid glass, you'd set the IOR to 1.5, but what do you set `thickness` to? Shouldn't it not be necessary?

Once that is clear it should be easy to create a workaround and/or see if it's a Blender issue.

**Comment by mockersf** [comment]
Thickness in gltf is defined in an extension: `KHR_materials_volume`
https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_volume/README.md

This is loaded by Bevy if it's available.

So you need to setup Blender to export materials with this extension. It can do it (https://github.com/KhronosGroup/glTF-Blender-IO/pull/1646) but I don't know Blender enough to know how to set it there.

**Comment by GitGhillie** [comment]
Yeah just to be clear, when I ask "How does one create a solid glass material in Bevy?" I'm talking about a Bevy material, nothing gltf related

**Comment by janhohenheim** [comment]
Quick googling for the Blender export settings days:

> Blender 3.3 allows you to add a glTF Material Output node, which includes values for occlusion, thickness, and specular – all the data needed for outputting volumes into a glTF file. When importing a glTF file, this node and associated values are automatically added and set.

> If you are creating an object from scratch and wish to output to glTF, you can enable the glTF Output Node in preferences: Preferences > Add-ons > Import-Export: glTF 2.0 format > Shader Editor Add-On

> Once enabled, the glTF Material Output Node will appear in the menu under Output Nodes.
